### DISCUSSION TITLE
[Trace Profiling] Duration and Self Duration in the Thread Stack are both 0ms

### DISCUSSION THREAD

**Discussion by BertOnline** [apache/skywalking#10550] [Q&A]
[Trace Profiling] Why is it that after analyzing, the Duration and Self Duration in the Thread Stack are both 0ms, and the previous 9.2.0 version is normal? Is it because of the problem with my sampling?

https://user-images.githubusercontent.com/17882970/225659178-549edab2-c07a-4bb0-b1a3-8abaebce76e9.mp4

**Comment by wu-sheng** [comment]
Check how many snapshots you collected from the agent. AFAIK, there is only one snapshot collected.
